src/accelerate/data_loader.py

```diff
@@ -354,11 +356,12 @@ class DataLoaderShard(DataLoader):
         - **total_dataset_length** (`int`) -- Total length of the inner dataset across all processes.
     """
 
-    def __init__(self, dataset, device=None, rng_types=None, synchronized_generator=None, **kwargs):
+    def __init__(self, dataset, device=None, rng_types=None, synchronized_generator=None, skip_batches=0, **kwargs):
         super().__init__(dataset, **kwargs)
         self.device = device
         self.rng_types = rng_types
         self.synchronized_generator = synchronized_generator
+        self.skip_batches = skip_batches
```

Comment:
```suggestion
        self.skip_n_batches = skip_n_batches
```
Suggestion based on previous comment